src/IndexRegistry.php

```diff
@@ -45,6 +45,28 @@ class IndexRegistry
      */
     protected static $options = [];
 
+    /**
+     * Fallback class to use
+     *
+     * @var string
+     */
+    protected static $fallbackClassName = Index::class;
+
+    /**
+     * Set fallback class name.
+     *
+     * The class that should be used to create a index instance if a concrete
+     * class for alias used in `get()` could not be found. Defaults to
+     * `Cake\ElasticSearch\Index`.
+     *
+     * @param string $className Fallback class name
+     * @return void
+     */
+    public static function setFallbackClassName($className)
```

Comment:
If we're going to add new public methods we should have tests, or we could break the behavior in future refactoring/cleanup work.

Reply:
Added IndexRegistryTest::testSetFallbackClassName